cafebabel/templates/articles/tags/detail.html

```diff
@@ -30,7 +30,7 @@
   {{ meta('name=twitter:url', request.url) }}
 {% endblock %}
 
-{% block body_class %}tag-page{% endblock %}
+{% block body_class %}tag-page {{ tag.slug }}{% endblock %}
```

Comment:
I think you should prefix the tag to avoid collisions with existing body classes (and associated styles).

Reply:
ok https://github.com/cafebabel/cafebabel.com/pull/397/commits/ad4d0d032fc3c75a005d57f428562f1effb8a601